src/components/BlockQuote/docs.md

```diff
@@ -0,0 +1,23 @@
+### `<BlockQuote />`
+
+A `<BlockQuote />` contains a `<BlockQuoteContent />` which itself can contain an arbitrary number of `<Editorial.P />` blocks.
+
+Optionally you can provide a `<FigureCaption />` to describe the quote source.
+
+
+```react|responsive
+<BlockQuote>
+  <BlockQuoteContent>
+    <Editorial.P>
```

Comment:
Does Editorial.P match the typography in the designs (I thought it's a sans-serif typography)? 

If not, I'd recommend to use (or add) the font cuts in components/Typography/styles.js and use those in some custom BlockQuoteParagraph component.

Reply:
attributes spread, got it.
And it is definitely not a sans-serif font, it just looked that way in my sketch file because of the missing fonts.

Why do you suggest against reusing `Editorial.P`?

Reply:
Looking at the exported designs, the blockquote text still looks sans-serif to me. But maybe I'm looking at old designs, let's follow up on Slack.